pkg/test/utils/kubernetes_integration_test.go

```diff
@@ -130,11 +130,22 @@ func TestRunCommand(t *testing.T) {
 	stdout = &bytes.Buffer{}
 
 	// assert background cmd returns process
-	cmd, err = RunCommand(context.TODO(), "", hcmd, "", stdout, stderr)
+	cmd, err = RunCommand(context.TODO(), "", hcmd, "", stdout, stderr, 0)
 	assert.NoError(t, err)
 	assert.NotNil(t, cmd)
 	// no stdout for background processes
 	assert.Empty(t, strings.TrimSpace(stdout.String()))
+
+	stdout = &bytes.Buffer{}
+	hcmd.Background = false
+	hcmd.Command = "sleep 42"
+
+	// assert foreground cmd times out
+	cmd, err = RunCommand(context.TODO(), "", hcmd, "", stdout, stderr, 2)
```

Comment:
Perhaps use `1` and save a second every time someone runs integration tests? :-)